scvi/models/distributions.py

```diff
@@ -0,0 +1,196 @@
+from typing import Union, Tuple
+import warnings
+
+import torch
+from torch.distributions import (
+    constraints,
+    Distribution,
+    Gamma,
+    Poisson,
+)
+from torch.distributions.utils import (
+    broadcast_all,
+    probs_to_logits,
+    lazy_property,
+    logits_to_probs,
+)
+
+from scvi.models.log_likelihood import log_nb_positive, log_zinb_positive
+
+
+def from_nb_params2_to_1(mu, theta, eps=1e-6):
```

Comment:
is there some way we can give these functions a more specific name? Also should we make them "private"? "_from..."

Reply:
tried my best but it's not that easy